src/main/flight/mixer.c

```diff
@@ -976,3 +970,26 @@ void filterServos(void)
 #endif
 }
 
+uint16_t pwmToMotor(uint16_t pwm)
+{
+    uint16_t motor = pwm;
+#ifdef USE_DSHOT
+    if (masterConfig.motorConfig.motorPwmProtocol == PWM_TYPE_DSHOT150 || masterConfig.motorConfig.motorPwmProtocol == PWM_TYPE_DSHOT600) {
+        motor = pwm == 1000 ? 0 : constrain((pwm - 977) * 2, 48, 2047);
```

Comment:
Yes it will, tested. 
I have all this rebased an ready to go. Either I make a new PR, or you pull my version from "https://github.com/AndersHoglund/betaflight.git play_with_dshot "
 
A new PR should probably be directed towards master...? or where do we stand with dev/master ?